shardingsphere-operator/api/v1alpha1/storage_node_types.go

```diff
@@ -17,7 +17,65 @@
 
 package v1alpha1
 
-import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type StorageNodePhaseStatus string
+
+const (
+	StorageNodePhaseReady    StorageNodePhaseStatus = "Ready"
+	StorageNodePhaseNotReady StorageNodePhaseStatus = "NotReady"
+)
+
+type StorageNodeConditionType string
+
+// StorageNodeConditionType shows some states during the startup process of storage node
+const (
+	StorageNodeConditionTypeAvailable StorageNodeConditionType = "Available"
```

Comment:
Please add explanation for these two conditions and update related design proposal.

Reply:
Just retain the "Available" condition for describe the whole storageNode is available or not, it's depends cluster is ready and all instances are ready.
We may add more additional conditions for more info in the future.

Reply:
> Just retain the "Available" condition for describe the whole storageNode is available or not, it's depends cluster is ready and all instances are ready. We may add more additional conditions for more info in the future.

Yeah, this makes sense.